File: Python/Program/Produk.py
class Produk:
    def __init__(self):
        self._nama_produk = ""
        self._brand = ""
        self._kode = ""
        self._harga = 0
        self._deskripsi = ""

    def get_nama_produk(self): return self._nama_produk
    def set_nama_produk(self, v): self._nama_produk = v

    def get_brand(self): return self._brand
    def set_brand(self, v): self._brand = v

    def get_kode(self): return self._kode
    def set_kode(self, v): self._kode = v

    def get_harga(self): return self._harga
    def set_harga(self, v): self._harga = v

    def get_deskripsi(self): return self._deskripsi
    def set_deskripsi(self, v): self._deskripsi = v

File: Php/Program/Produk.php
<?php
class Produk {
    private string $namaProduk, $brand, $kode, $deskripsi, $foto;
    private float $harga;

    public function __construct(string $namaProduk, string $brand, string $kode, float $harga, string $deskripsi, string $foto){
        $this->namaProduk=$namaProduk; $this->brand=$brand; $this->kode=$kode; $this->harga=$harga;
        $this->deskripsi=$deskripsi; $this->foto=$foto;
    }
    public function getNamaProduk(){return $this->namaProduk;}
    public function setNamaProduk($v){$this->namaProduk=$v;}
    public function getBrand(){return $this->brand;}
    public function setBrand($v){$this->brand=$v;}
    public function getKode(){return $this->kode;}
    public function setKode($v){$this->kode=$v;}
    public function getHarga(){return $this->harga;}
    public function setHarga($v){$this->harga=$v;}
    public function getDeskripsi(){return $this->deskripsi;}
    public function setDeskripsi($v){$this->deskripsi=$v;}
    public function getFoto(){return $this->foto;}
    public function setFoto($v){$this->foto=$v;}
}
